Makefile

```diff
@@ -22,6 +22,14 @@ LDFLAGS=-ldflags "-w -s -X main.Version=${TAG}"
 setup: setup_githooks
 	@echo Installing application dependencies:
 	dep ensure
+	make build_swagger_api
+
+.PHONY: build_swagger_api
+build_swagger_api:
+	@echo Installing Swagger dep:
+	go get -u github.com/go-swagger/go-swagger/cmd/swagger
+	@echo Updating Swagger api:
+	go generate $(APP_FILE)
```

Comment:
This command will run all commands described by directives in our code. Your new swagger command is the only one in our code for now. If there is more in the future, this will end up running those as well.

Reply:
It is the same way that is done in the [aerogear-app-metrics](https://github.com/aerogear/aerogear-app-metrics/blob/master/cmd/metrics-api/metrics-api.go). Following some important info.

* The command should be called always in order to ensure that the swagger API will be updated/accurate. 
* In order to generate the swagger, API is required to execute it from the main.go directory. 

IMHO, for now, at least shows fine we use the `go generate` in order to achieve the expected result here. However, if you do not agree how would you like to see it implemented it? Have you any suggestion or could we move forward in this way then if be required we can change it in future?

Reply:
Is it possible to run the swagger command directly from the make file? `swagger generate spec -m -o ../../api/swagger.yaml`

Reply:
It was changed as you requested. 👍 Thank you for the input.